shared/feature/profile/src/main/java/com/epmedu/animeal/profile/domain/model/Region.kt

```diff
@@ -0,0 +1,595 @@
+@file:Suppress(
+    "ComplexMethod",
+    "LongMethod",
+    "ReturnCount"
+)
+
+package com.epmedu.animeal.profile.domain.model
```

Comment:
What do you think about moving this file to the `common` module?

Reply:
For now looks quite logical for me, as we have signup feature depending on profile

Reply:
Makes sense